content/applications/sales/point_of_sale/payment_methods/terminals/vantiv.rst

```diff
@@ -1,45 +0,0 @@
-======
-Vantiv
```

Comment:
Need to add a 301 redirect for the deleted file, to the `redirects.txt` file (can point to `terminals.rst`)